<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {border-image: url(:/m1.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labIP">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>监听地址</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labPort">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>端口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="boxIP"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="boxPort">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="textInfo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="monitorState">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>监听状态</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="socketState">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 0)</string>
        </property>
        <property name="text">
         <string>socket状态</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="textMonitorState">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="textSocketState">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="btnSend">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);
background:transparent;</string>
        </property>
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
color:rgb(255, 255, 255);</string>
      </property>
      <property name="plainText">
       <string>
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>317</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:rgb(255, 255, 0);
font: 700 9pt &quot;Times New Roman&quot;;
</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actFinish"/>
   <addaction name="actClear"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actStart">
   <property name="text">
    <string>开始监听</string>
   </property>
  </action>
  <action name="actFinish">
   <property name="text">
    <string>停止监听</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
